iOS/Flows/Home/Moments/Capture/PIP/PiPRecordingViewController.swift

```diff
@@ -135,9 +136,15 @@ class PiPRecordingViewController: ViewController, AVCaptureVideoDataOutputSample
     // MARK: - PUBLIC
     
     func startRecording() {
-        let settings = self.backOutput.recommendedVideoSettingsForAssetWriter(writingTo: .mp4)
-        let audioSettings = self.micDataOutput.recommendedAudioSettingsForAssetWriter(writingTo: .mp4)
```

Comment:
These videos are .movs, so I changed these. It doesn't make a huge difference though.